torchvision/models/detection/transform.py

```diff
@@ -122,20 +138,24 @@ def _onnx_batch_images(self, images, size_divisible=32):
         return torch.stack(padded_imgs)
 
     def batch_images(self, images, size_divisible=32):
-        if torchvision._is_tracing():
-            # batch_images() does not export well to ONNX
-            # call _onnx_batch_images() instead
-            return self._onnx_batch_images(images, size_divisible)
-
-        max_size = tuple(max(s) for s in zip(*[img.shape for img in images]))
-        stride = size_divisible
-        max_size = list(max_size)
+        # type: (List[Tensor], int)
+        if not torch.jit.is_scripting():
+            if torchvision._is_tracing():
```

Comment:
you don't need the `if not torch.jit.is_scripting` here because `torchvision._is_tracing` returns False in script